audition/distance_from_best.py

```diff
@@ -33,11 +33,23 @@ def _create(self):
             train_end_time timestamp,
             metric text,
             parameter text,
-            raw_value float,
-            below_best float,
-            below_best_next_time float
+            dist_from_abs_worst float,
```

Comment:
I think I see what you're after with `dist_from_abs_worst` but I'm not sure that's always going to be well-defined and we probably still want to have the raw value around regardless.